.env.sample

```diff
@@ -8,3 +8,5 @@ DT_RUM_URL=
 SESSION_SECRET=dev_secret
 ENABLE_PREVIEW="development"
 LANGUAGE_TOGGLE=false
+SPINNER_REQUEST_INTERVA=3000
```

Comment:
`INTERVAL` ?
and could we have these prefixed with `MAM` as we'll want to be able to configure the values for the DAD screen separately